packages/salesforcedx-vscode-core/test/vscode-integration/commands/functions/forceFunctionInvokeCodeLensProvider.test.ts

```diff
@@ -13,7 +13,7 @@ import { getRootWorkspacePath } from '../../../../src/util';
 describe('Force Function Invoke Codelens', () => {
   it('should create codelens for a json payload file', async () => {
     const cancellationTokenSource = new CancellationTokenSource();
-    workspace
+    return workspace
```

Comment:
fyi @jag-sfdc  for async tests, we need to return this promise otherwise the tests won't fail properly.